core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,11 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+  .popover {
```

Comment:
Sorry, I don't know, and I can't look into that now, but you're right that if there are conflicts the person whose PR gets merged later must handle them. This is a normal part of shared repository dev workflows so I wouldn't worry too much about it -- but if your PRs overlap then you should coordinate and figure out what to do.